playground/app/pages/index.vue

```
@@ -47,6 +47,25 @@ const providers = computed(() =>
       disabled: Boolean(user.value?.bluesky),
       icon: 'i-simple-icons-bluesky',
     },
+    {
+      title: user.value?.mastodon || 'Mastodon',
+      onClick() {
+        const instance = prompt('Enter your Mastodon instance (e.g. mastodon.social)')
+        if (instance) {
+          navigateTo(
```

ESLint

[error] 'navigateTo' is not defined.

(no-undef)